test/smoke/src/helpers/smokeTestsConstants.ts

```diff
@@ -11,10 +11,12 @@ public static defaultTargetAndroidDeviceName = "emulator-5554";
 public static smokeTestSetupAwaitTimeout = 10 * 60 * 1000;
 // Timeout for emulator boot
 public static emulatorLoadTimeout = 90 * 1000;
-// Timeout for Spectron to wait for UI elements response to interaction
-public static spectronElementResponseTimeout = 30 * 1000;
+// Timeout for Spectron to wait for UI elements response to interaction (in seconds)
+public static spectronElementResponseTimeout = 250;
```

Comment:
Why can't we use the same format everywhere in the tests? It'd help us to avoid confusion and need to go to variable declaration everytime to check what format it uses.

Reply:
I agree, but seems that Spectron gets waitTime in seconds

Reply:
> I agree, but seems that Spectron gets waitTime in seconds

Okay, then.